### ISSUE TITLE
Multiselect field missing dropdown caret and cross icon

### ISSUE THREAD

**Issue by sachajudd** [silverstripe/silverstripe-ckan-registry#86]
Just raising my concern about using a mulitselect field that doesn't have a caret (or remove cross icon) to indicate that it also has a dropdown. @ScopeyNZ explained potentially because this has text auto complete. I also have concerns about the placeholder text "Select Some Options" and also the contrast for accessibility purposes. We discussed that this is a bug in framework and to raise an issue there. 

![image](https://user-images.githubusercontent.com/24258161/50810938-06555a80-1371-11e9-89fd-5e7de8585d5d.png)

![image](https://user-images.githubusercontent.com/24258161/50810944-0bb2a500-1371-11e9-8f23-0d75ef4faae6.png)

[Designs](https://projects.invisionapp.com/share/8GOKGM7BTR5#/screens/325695106_Data_Module_CKAN_-_Filter_Text_Type_Modal) for what we envisioned.

In the meantime for CKAN I think it's worth talking about some alternatives we could use such as using something like tagfield which has a dropdown caret. 

Any thoughts welcome.

All form designs can be found in our [Styleguide](https://projects.invisionapp.com/dsm/silver-stripe/silver-stripe/folder/components/5aa5dcf342ed8d001136a1d7)

**Comment by robbieaverill** [comment]
>We discussed that this is a bug in framework and to raise an issue there which I'll add this week.

Yeah, I think that's where we should change this pattern. At this stage we're using existing form fields from core, I don't think there's a reason to create new ones - we should update the core ones if needed (being mindful that they'd flow on to all places they're used - which is a good thing I guess).

>In the meantime for CKAN I think it's worth talking about some alternatives we could use such as using something like tagfield which has a dropdown caret and cross icon.

To be honest, introducing an extra module dependency in order to achieve this seems pretty low value to me. I think we'd be better to change the multi select design in core instead.

FWIW the core form field is `ListboxField` here

**Comment by sachajudd** [comment]
Yeah agreed, I'll cc @clarkepaul into this for now to have a look at.
